AUTHORS

```diff
@@ -68,3 +68,5 @@ The University of Washington
 Tobin Harding <me@tobin.cc>
 Vladimír Chalupecký <vladimir.chalupecky@gmail.com>
 Yevgeniy Vahlis <evahlis@gmail.com>
+Saran Ahluwalia <ahlusar.ahluwalia@gmail.com>
```

Comment:
Sorry @ahlusar1989 , please add yourself alphabetically in the list (after Sam Zaydel)